.github/workflows/release-prep.yml

```
@@ -48,38 +60,48 @@ jobs:
           sed -i "s/minor: int = \([0-9]\+\)/minor: int = $MINOR/g" heat/core/version.py
           sed -i "s/micro: int = \([0-9]\+\)/micro: int = $MICRO/g" heat/core/version.py
           sed -i "s/extension: str = .*/extension: str = None/g" heat/core/version.py
+          
+          { echo -e "# v${MAJOR}.${MINOR}.${MICRO} - ${{github.event.inputs.title}}\n${{ steps.release_drafter.outputs.body}}\n"; cat CHANGELOG.md; } > tmp.md
+          mv tmp.md CHANGELOG.md
 
           # Git configuration with anonymous user and email
           git config --global user.email ""
           git config --global user.name "Heat Release Bot"
 
           # Commit the changes
-          git add heat/core/version.py
+          git add heat/core/version.py CHANGELOG.md
           git commit -m "Bump version to $VERSION"
 
           # Push the changes
           git push --set-upstream origin pre-release/${VERSION}
 
           # Create PR for release
           gh pr create \
-          --base release \
+          --base stable \
           --head ${{ env.PR_BRANCH }} \
           --title "Heat ${{ env.VERSION }} - Release" \
           --body "Pre-release branch for Heat ${{ env.VERSION }}.
-          Any release work should be done on this branch, and then merged into the release branch and main, following git-flow.
+          Any release work should be done on this branch, and then merged into the release branch and main, following git-flow guidelines.
 
           TODO:
           - [x] Update version.py
-          - [ ] update the Requirements section on README.md if needed
-          - [ ] Update CITATION.cff if needed
-          - [ ] Ensure the Changelog is up to date
+          - [ ] Ensure Citation file is up to date.
+          - [ ] Ensure the Changelog is up to date.
 
-          [1]: https://github.com/peter-evans/create-pull-request" \
-          --label invalid
+          DO NOT DELETE BRANCH AFTER MERGING!" \
+          --label "pre-release"
 
           # Create PR for main
           gh pr create --base main \
           --head ${{ env.PR_BRANCH }} \
           --title "Heat ${{ env.VERSION }} - Main" \
-          --body "Copy of latest pre-release PR targeting release." \
-          --label invalid
+          --body "Copy of latest pre-release PR targeting release. 
+          DO NOT CHANGE ANYTHING UNTIL THE RELEASE BRANCH HAS BEEN MERGED. 
```

```suggestion
          --body "Copy of latest pre-release PR targeting release. 
          DO NOT CHANGE ANYTHING UNTIL "Heat ${{ env.VERSION }} - Release" HAS BEEN MERGED. 
```

not sure this will work as intended. It would be great to reference the PR number directly, if it isn't too complicated.